### DISCUSSION TITLE
how to set the plugin's loglevel to debug?

### DISCUSSION THREAD

**Discussion by linxd** [nextgenhealthcare/connect#5599] [General]
I want to debug some plugin, how to set the log4j2.properties?

Thanks

**Comment by jonbartels** [comment]
What plugin do you want to debug?

You can just add lines to `log4j.properties` like:
```
log4j.logger.java.sql = ERROR
log4j.logger.com.your.plugin = ALL
```

**Reply by linxd** [reply]
in conf dir, there only have log4j2.properties. should  add a log4j.properties?
have added ,but still no efffect.

**Reply by jonbartels** [reply]
Can you please share your log4j.properties file?

What plugin are you trying to get logs for?

**Reply by linxd** [reply]
logger.clustering.name = com.mirth.connect.plugins.clustering
logger.clustering.level = WARN
logger.clusteringadvanced.name = com.mirth.connect.plugins.clusteringadvanced
logger.clusteringadvanced.level = WARN
logger.clusteringbasic.name = com.mirth.connect.plugins.clusteringbasic
logger.clusteringbasic.level = WARN
